From fed6abf92f25d1a64a5cc36f5db022651eb79ab7 Mon Sep 17 00:00:00 2001
Message-ID: <fed6abf92f25d1a64a5cc36f5db022651eb79ab7.1705743628.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1705743627.git.sharlatanus@gmail.com>
References: <cover.1705743627.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Tue, 16 Jan 2024 23:35:39 +0000
Subject: [PATCH 18/38] gnu: go-github-com-rfjakob-eme: Move to (gnu packages
 golang-crypto).

* gnu/packages/golang.scm (go-github-com-rfjakob-eme): Move from here ...
* gnu/packages/golang-crypto.scm: ... to here.

Change-Id: Ic38d75fd5d3114e246c08abe881fae62e6f9c6b9
---
 gnu/packages/golang-crypto.scm | 22 ++++++++++++++++++++++
 gnu/packages/golang.scm        | 23 -----------------------
 2 files changed, 22 insertions(+), 23 deletions(-)

diff --git a/gnu/packages/golang-crypto.scm b/gnu/packages/golang-crypto.scm
index 3bcf86673d..3057e803ba 100644
--- a/gnu/packages/golang-crypto.scm
+++ b/gnu/packages/golang-crypto.scm
@@ -373,6 +373,28 @@ (define-public go-github-com-refraction-networking-utls
 library merely changes ClientHello part of it and provides low-level access.")
     (license license:bsd-3)))
 
+(define-public go-github-com-rfjakob-eme
+  (package
+    (name "go-github-com-rfjakob-eme")
+    (version "1.1.2")
+    (source
+     (origin
+       (method git-fetch)
+       (uri (git-reference
+             (url "https://github.com/rfjakob/eme")
+             (commit (string-append "v" version))))
+       (file-name (git-file-name name version))
+       (sha256
+        (base32 "1yrbhvy0337mf12fp8p4sy8ry8r3w2qfdf8val5hj07p2lri0cqk"))))
+    (build-system go-build-system)
+    (arguments
+     '(#:import-path "github.com/rfjakob/eme"))
+    (home-page "https://github.com/rfjakob/eme")
+    (synopsis "EME for Go")
+    (description "EME (ECB-Mix-ECB or, clearer, Encrypt-Mix-Encrypt) is a
+wide-block encryption mode developed by Halevi and Rogaway.")
+    (license license:expat)))
+
 (define-public go-github-com-riobard-go-bloom
   (let ((commit "cdc8013cb5b3eb0efebec85f0e904efccac42df9")
         (revision "0"))
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 37edfd7eb0..c8b211441c 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -1655,29 +1655,6 @@ (define-public go-github-com-pkg-xattr
 .")
     (license license:bsd-2)))
 
-(define-public go-github-com-rfjakob-eme
-  (package
-    (name "go-github-com-rfjakob-eme")
-    (version "1.1.2")
-    (source (origin
-              (method git-fetch)
-              (uri (git-reference
-                    (url "https://github.com/rfjakob/eme")
-                    (commit (string-append "v" version))))
-              (file-name (git-file-name name version))
-              (sha256
-               (base32
-                "1yrbhvy0337mf12fp8p4sy8ry8r3w2qfdf8val5hj07p2lri0cqk"))))
-    (build-system go-build-system)
-    (arguments
-     '(#:import-path "github.com/rfjakob/eme"))
-    (home-page "https://github.com/rfjakob/eme")
-    (synopsis "EME for Go")
-    (description
-     "EME (ECB-Mix-ECB or, clearer, Encrypt-Mix-Encrypt) is a wide-block encryption
-mode developed by Halevi and Rogaway.")
-    (license license:expat)))
-
 (define-public go-github-com-schachmat-ingo
   (package
     (name "go-github-com-schachmat-ingo")
-- 
2.41.0

